package.json

```
@@ -3,7 +3,7 @@
     "version": "0.0.1",
     "publisher": "vsciot-vscode",
     "displayName": "Azure IoT Tools",
-    "description": "The ultimate collection of extensions for working with Azure IoT in VS Code!",
+    "description": "Manage IoT devices, connect them to cloud, deploy AI into IoT Edge and more, all on Azure IoT, all from VS Code, in this one extension from Microsoft.",
```

why do we want to call out AI on Edge? And shall we mention functionalities from Toolkit? can we change it to:
```suggestion
    "description": "Manage IoT devices & Edge, connect them to cloud, Azure IoT services management and more, all on Azure IoT, all from VS Code, in this one extension from Microsoft.",
```